server/static/static.go

```diff
@@ -193,6 +180,34 @@ func serveIndexTemplate(ctx context.Context, env environment.Env, tpl *template.
 		PopupAuthEnabled:                       *popupAuthEnabled,
 	}
 
+	if auth := env.GetAuthenticator(); auth != nil {
```

Comment:
Why not just leave these in place above? I find the current approach much more readable.

Reply:
i agree, even though it's repetitive.

Reply:
Getting rid of this.